about.html

```diff
@@ -0,0 +1,147 @@
+                <img id= "close-menu" src="./capstoneimgs/x-button-icon-20.jpg" alt="" class="close-menu-icon p-3">
+              </div>
+              <div id="menu-container">
+                <ul class="menu">
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center text-danger" >
+                    <a href="index.html">Home</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center"> 
+                    <a href="#program">Program</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#join">Join</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#sponsor">Sponsor</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#news">News</a>
+                  </li>
+              </ul>
+              </div> 
+            </nav>
+          </div>                     
+        </header>
+  </section>
+
+
+      <section id="headline">
+        <p class="headline-hello">"Hello! World Music Fans!"</p>
+          <h1 class="headline-title">INTI RAYMI MUSIC FESTIVAL 2023</h1>
+        <p class="headline-p">Inti Raymi Music Festival brings together some of the most recognized musicians in Ecuador to perform for you all in the Mitad del Mundo City's Central Plaza. Come experience this celebration of thousands of years of tradition in cultural heritage.</p>
+        <p class="contact">Please contact us per email for any further questions 
+          about Inti Raymi Music Festival 2023!</p>
+        <address>intiraymifestivalecuador@somemail.not</address>
+      </section>
+    </header>
+    <main>
+      <section id="creative">
+        <h2 class="sectionTitle">Inti Raymi Music Festival 2023 Logo</h2>
+        <p class="creative-p">The logo of Inti Raymi Music Festival 2023 was decided through a logo competition by World-Music International Radio in July the 11th, 2022.</p>
+        <img src="./capstoneimgs/logo-large.png" alt="World Music Festival logo">
+      </section>
+      <section id="history">
+        <h1 class="sectionTitle">See the past Inti Raymi Music Festivals</h1>
+        <p class="history-p">Take a look at the past two Inti Raymi Music Festivals that took place in La Paz and in Lima.</p>
+        <div class="lapaz-lima">
+        <a href="#" class="ir2022 hist-link">
+          <article>
+            <time class="hist-time">2022</time>
+            <h1 class="hist-title">Inti Raymi Music Festival 2022</h1>
+          </article>
+        </a>
+        <a href="#" class="ir2020 hist-link">  
+          <article>
+            <time class="hist-time">2020</time>
+            <h1 class="hist-title">Inti Raymi Music Festival 2021</h1>
+          </article>
+        </a>
+      </div>
+      </section>
```

Comment:
- [ ] Please consider adding a "divider" border to each section so it matches the design! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/226497228-e5119e4f-2320-45d3-8585-01ef5b1c0424.png) | ![image](https://user-images.githubusercontent.com/77069011/226497253-937a44c0-d2c0-4340-b325-e4255e6f2cfb.png) | 

- [ ] And the same happens here with the spacing between the title and the content of each section! Please fix it. 🤓 🔧

Reply:
Thank you, it is done!

Reply:
- [ ] This issue is well-resolved :heavy_check_mark: :